tensorflow/python/keras/backend.py

```diff
@@ -1679,10 +1679,10 @@ def zeros_like(x, dtype=None, name=None):
   Example:
 
 
-  from tensorflow.keras import backend as K
-  kvar = K.variable(np.random.random((2,3)))
-  kvar_zeros = K.zeros_like(kvar)
-  K.eval(kvar_zeros)
+  >>> from tensorflow.keras import backend as K
```

Comment:
Please instead use the ``` code block formatting syntax since these lines are not meant to be executed as part of our "doctest" suite (testable code examples in docstrings).

Reply:
Done!